### PR TITLE
sst_import: support multiple ranges in a single switch mode request

### PR DESCRIPTION
**Pull request by SpadeA-Tang**
### What is changed and how it works?

Issue Number: Close https://github.com/tikv/tikv/issues/14916

What's Changed:

```commit-message
support multiple ranges in a single switch mode request
```

### Related changes

https://github.com/pingcap/kvproto/pull/1148

### Check List 

Tests 

- Unit test
- Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -200,8 +200,8 @@ procinfo = { git = "https://github.com/tikv/procinfo-rs", rev = "6599eb9dca74229
 # When you modify TiKV cooperatively with kvproto, this will be useful to submit the PR to TiKV and the PR to
 # kvproto at the same time.
 # After the PR to kvproto is merged, remember to comment this out and run `cargo update -p kvproto`.
-# [patch.'https://github.com/pingcap/kvproto']
-# kvproto = { git = "https://github.com/your_github_id/kvproto", branch = "your_branch" }
+[patch.'https://github.com/pingcap/kvproto']
+kvproto = { git = "https://github.com/SpadeA-Tang/kvproto", branch = "import-sstss" }
```
Comment (overvenus) [inline Cargo.toml:204]: Please comment it out and update pingcap/kvproto.

**Review by overvenus — APPROVED** [review]

**Review by bufferflies — APPROVED** [review]

**Pull request merged**